torch/csrc/jit/codegen/cuda/lower_expr_sort.cpp

```diff
@@ -769,10 +828,52 @@ bool ExprSegmentationSorter::supportedMerge(ExprGroup* sg1, ExprGroup* sg2) {
     return false;
   }
 
+  if (GpuLower::current()->caLoopMap().areMapped(
```

Comment:
This branch does not seem to have any effect.

Reply:
Thanks, was left over from debug printing.